### ISSUE TITLE
Error Code 2: Internal Error (L__self___conv1.weight: Quantized constant is only allowed before DQ or PLUGIN_V2 node)

### ISSUE THREAD

**Issue by junstar92** [NVIDIA/TensorRT#3919]
I try to use **Plugin V3** with quantized input and weights. But, the following error was encountered:

```
Error[2]: [qdqGraphOptimizer.cpp::matchQuantizedConstantPluginOrDQ::4004] Error Code 2: Internal Error (L__self___conv1.weight: Quantized constant is only allowed before DQ or PLUGIN_V2 node)
```

It seems that the quantized constant should be allowed before the PLUGIN_V3 node, but it was missed when PLUGIN_V3 was updated in TensorRT 10.0.

There is no issue when changing Plugin V3 to Plugin V2. It needs to add Plugin V3 to list of nodes that can receive quantized weights as input.

## Environment

**TensorRT Version**: TensorRT 10.0.1
**NVIDIA GPU**: NVIDIA RTX 4000 Ada
**NVIDIA Driver Version**: 
**CUDA Version**: 12.4
**CUDNN Version**:

**Comment by lix19937** [comment]
Plugin V3 relates to `data-dependent output shapes`, see https://github.com/NVIDIA/TensorRT/tree/release/10.0/samples/sampleNonZeroPlugin#implementing-a-nonzero-plugin-using-ipluginv3-interface

**Comment by junstar92** [comment]
It is fixed in V10.1.
